test/3rdParty/xfail.json

```diff
@@ -9,12 +9,8 @@
             "38": "skip",
             "39": "skip",
             "40": "skip",
-            "43": true,
-            "44": true,
-            "45": true,
-            "46": true,
-            "47": true,
-            "48": true,
+            "47": "skip",
```

Comment:
As far as I can remember, 47 and 48 did hang the tests, I guess your patch in #161 deals with that now.